client/lib/plans/constants.js

```diff
@@ -354,13 +354,13 @@ const getPlanBusinessDetails = () => ( {
 	type: TYPE_BUSINESS,
 	getTitle: () => i18n.translate( 'Business' ),
 	getAudience: () => i18n.translate( 'Best for small businesses' ),
-	getBlogAudience: () => i18n.translate( 'Best for brands' ),
+	getBlogAudience: () => i18n.translate( 'Best for small businesses' ),
 	getPortfolioAudience: () => i18n.translate( 'Best for small businesses' ),
 	getStoreAudience: () => i18n.translate( 'The plan for small businesses' ),
 	getDescription: abtest => {
 		if ( abtest && abtest( 'businessPlanDescriptionAT' ) === 'pluginsAndThemes' ) {
 			return i18n.translate(
-				'{{strong}}Best for Small Business:{{/strong}} Power your' +
+				'{{strong}}Best for Small Businesses:{{/strong}} Power your' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **20** times:
`translate( '{{strong}}Best for Small Business:{{/strong}} Power your business website with custom plugins and themes, unlimited premium and business theme templates, Google Analytics support, unlimited storage, and the ability to remove WordPress.com branding.' )` *ES Score: 20*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).